public/lessons/www/http.md

```diff
@@ -7,6 +7,14 @@ HTTP stands for **Hypertext Transfer Protocol**. In a nutshell, an HTTP request
 
 ![](http://betterexplained.com/wp-content/uploads/compression/HTTP_request.png ""HTTP Request"")
```

Comment:
I know that in markdown the brackets mean "alt" and the quotes mean "title" but I have two questions:

1. For screenreaders/ARIA it cool to have a title but no alt?
2. Why the double-double quotes?

https://daringfireball.net/projects/markdown/syntax#img

Reply:
1. I think both are important. I can adjust this. 
2. I don't know why I did doubles here.